cranelift/codegen/src/isa/x64/lower.isle

```diff
@@ -2318,6 +2318,10 @@
 (rule (lower (has_type $F64X2 (sqrt x)))
       (x64_sqrtpd x))
 
+;; Rules for `fpromote` ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(rule (lower (has_type $F64 (sqrt x)))
```

Comment:
Should this be `fpromote` rather than `sqrt` here?

Reply:
Strange .. running the wrong test.